app/src/main.js

```diff
@@ -6,10 +6,17 @@ import App from './App.vue'
 import './registerServiceWorker'
 import store from './store'
 import router from './router'
+import VueSocketIO from 'vue-socket.io'
+import SocketIO from 'socket.io-client'
```

Comment:
We shouldn't need this if we are using vue-socket.io. We can create a TD ticket for this fix. At implementation, Atul was just playing with different scenarios but didn't have enough time to fix those before he left.